src/hotfixes/runtime-2900-lazy-migrate-remove-local-assets.ts

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
_(note from author)_

While locally I ran `npx prettier --check --ignore-path .prettierignore '**/*.(yml|js|ts|json)'` and it didn't complain, seems I am missing something since in the CI still doesn't work.

I can revert these changes and leave it as is for now.